### DISCUSSION TITLE
Database support by Stackstorm

### DISCUSSION THREAD

**Discussion by amitgupta369** [StackStorm/st2#5801] [General]
Hi,

I am planning to integrate stackstorm with couchdb or mysql. is it possible to change the default database used by stackstorm(mongodb) to couchdb or mysql.

Thanks in advance.

**Comment by arm4b** [comment]
No, MongoDB is the only supported DB for StackStorm.
https://github.com/StackStorm/st2/issues/3778 is also a relevant discussion.
